src/uct/sm/mm/mm_iface.c

```diff
@@ -403,8 +403,12 @@ static void uct_mm_iface_recv_messages(uct_mm_iface_t *iface)
         ret = recvfrom(iface->signal_fd, &sig, sizeof(sig), 0, NULL, 0);
         if (ret == sizeof(sig)) {
             ucs_debug("mm_iface %p: got connect message", iface);
-            ucs_callbackq_add_safe(&iface->super.worker->progress_q,
-                                   uct_mm_iface_progress, iface);
+            if (iface->super.prog.refcount++ == 0) {
```

Comment:
so this will be changed to calling uct_worker_progress_register_safe with some flag in the next PR?